js/ide.js

```diff
@@ -2127,6 +2127,29 @@ var ide = new function() {
         encodeURIComponent(query) +
         "&target=compact";
 
+      function constructOverpassQLUrl(query) {
+        // remove /* */ comments from query
+        query = query.replace(/\/\*(.|\n)*?\*\//g, "");
```

Comment:
I believe using `[\S\s]` (i.e. _whitespace_ or _not whitespace_) is better to match multiline strings in javascript (your approach wouldn't work with `\r` newlines or other exotic stuff that doesn't match the `.` in js. see https://stackoverflow.com/a/1981692/1627467 or https://www.regular-expressions.info/dot.html for further info

Reply:
I didn't knew about \r! thanks